src/main/java/seedu/address/logic/commands/SellCommand.java

```diff
@@ -56,8 +58,8 @@ public CommandResult execute(Model model) throws CommandException {
 
         decreaseQuantity(model, soldGood);
         model.commit();
-        return new CommandResult(String.format(MESSAGE_SUCCESS,
-                soldGood.getGoodQuantity().goodQuantity, soldGood.getGoodName().fullGoodName));
+        return new CommandResult(String.format(MESSAGE_SUCCESS, soldGood.getGoodQuantity().goodQuantity,
```

Comment:
I have implemented the transaction history, maybe should add a new buy transaction and add it to the transaction history?

Reply:
Having the sell command add the corresponding sell transaction to the transaction history was the planned next enhancement after this one if approved.